database/migrations/2020_05_04_150000_create_bookings_table.php

```diff
@@ -16,22 +16,21 @@ public function up()
         Schema::create('bookings', function (Blueprint $table) {
             $table->id();
             $table->string('slot_number');
+            $table->foreignIdFor(app('user'));
+            $table->morphs('order'); // Can this be $table->foreignIdFor(app('order'));
             $table->integer('total_participants');
             $table->unsignedInteger('total_amount');
             $table->unsignedInteger('amount');
             $table->unsignedInteger('total_taxes');
             $table->unsignedInteger('total_fees');
             $table->morphs('experience');
-            $table->morphs('order');
+            $table->morphs('subject');
             $table->foreignIdFor(Rate::class);
             $table->foreignIdFor(BookingCategory::class);
-            $table->unsignedBigInteger('booking_status_id');
-            $table->foreign('booking_status_id')->references('id')->on('booking_status');
```

Comment:
I removed these since we are using tipoff/statuses now for booking status

Reply:
@sl0wik Or is this what you are using for the current Status of the Booking?

Reply:
Booking status can be removed.